ocaml/src/albamgr_protocol.ml

```diff
@@ -113,8 +113,12 @@ module Protocol = struct
     module Update = struct
       open Nsm_model.OsdInfo
       type t = {
+        (* asd/kinetic *)
```

Comment:
we might have been lifting out commonalities too early. Now with Alba osds, this is working against us.